README.md

```diff
@@ -208,6 +212,50 @@ which can be modified, or additional environments can be added by just adding a
 new YAML stanza with the right keys; this new environment can then be used with
 the `--environment` flag when running `search_solr_tools harvest`.
 
+#### Logging
+
+By default, when running the harvest, harvest logs are written to the file
+`/var/log/search-solr-tools.log` (set to `warn` level), as well as to the console
+at `info` level.  These settings are configured in the `environments.yaml` config
+file.  While these settings are configured in the `common` section of that file,
+they can be overridded for specific environments by adding the configuration
+setting to the individual environment section.  **NOTE: The environment is based
```

Comment:
As per our discussion, I think this should be "the `environment` setting indicates the environment used for harvesting..."